test/git_test.py

```diff
@@ -2,8 +2,12 @@
 # pylint: disable=redefined-outer-name
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import pathlib
+import sys
```

Comment:
Almost there! It looks like the CI checks caught this unused `import sys` and the one in `cola/git.py` as well.

Thanks for taking these on.

Reply:
> unused import

Ah, yep, I missed that. A pre-commit hook for flake8 can catch that, too. I'll introduce brunette, flake8, and text standardization (end-of-file newlines and trailing whitespace) in a separate PR.

Reply:
Jiminy Christmas, I need pre-commit hooks to develop effectively!

I've fixed another issue caught by pylint in CI. Probably the pre-commit hook needs to be pylint, not flake8.

Reply:
Update: #1333 introduces a [pre-commit](https://pre-commit.com/) config so style can be checked and enforced prior to allowing developers to even commit their changes. Brunette and pylint are part of that config. :grin: